app/managers.py

```diff
@@ -0,0 +1,62 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self, db_path: str) -> None:
+        self.connection = sqlite3.connect(db_path)
+        self.cursor = self.connection.cursor()
+        self._create_table()
+
+    def _create_table(self) -> None:
+        self.cursor.execute("""
+            CREATE TABLE IF NOT EXISTS actor (
+                id INTEGER PRIMARY KEY,
+                first_name VARCHAR(255) NOT NULL,
+                last_name VARCHAR(255) NOT NULL
+            )
+        """)
+        self.connection.commit()
+
+    def create(self, first_name: str, last_name: str) -> None:
+        if not first_name or not last_name:
+            raise ValueError("First name and last name cannot be empty")
+
+        self.cursor.execute("""
+            INSERT INTO actor (first_name, last_name)
+            VALUES (?, ?)
+        """, (first_name, first_name))
+        self.connection.commit()
+
+    def all(self) -> list[Actor]:
+        self.cursor.execute("""
+            SELECT id, first_name, last_name
+            FROM actor
+        """)
+        rows = self.cursor.fetchall()
```

Comment:
no need to save trow variable you can pull out data from cursor in list compreneshiocomprehensionn directly